core-bundle/src/Resources/contao/languages/en/tl_undo.xlf

```diff
@@ -36,7 +36,7 @@
         <source>Restore element ID %s</source>
       </trans-unit>
       <trans-unit id="tl_undo.parent_modal">
-        <source>Show parent of %s ID %s</source>
+        <source>Show parent element of %s ID %s</source>
```

Comment:
Shouldn't this be "parent **record**"?

Reply:
To me, "show parent record" is this:

<img width="901" alt="" src="https://user-images.githubusercontent.com/1192057/149291827-061867a8-58a1-4ac3-ab5e-1cff14356446.png">

So in my opinion, the link should say "Go to the parent […]" instead of "Show the parent […]". Im not sure what to use for […], though.

Reply:
To me, "show parent record" is this:

<img width="881" alt="" src="https://user-images.githubusercontent.com/1192057/149292440-f25deab2-7642-49dd-b6f4-c069006819c3.png">

So in my opinion, the link should say "Go to the parent […]" instead of "Show the parent […]". Im not sure what to use for […], though.

Reply:
I see your point with `go` vs `show`. Maybe we should use `origin` rather than parent? In the end we show the view where the deleted record would be shown if it had not been removed. 
So `Go to origin of %s ID %s`?

Reply:
That's good. 👍

Reply:
Added in 5ed440f7ddb5e81eef45f41119f5319359975b03.